### DISCUSSION TITLE
bulk actions on grids

### DISCUSSION THREAD

**Discussion by ErikNoort** [minvws/nl-kat-coordination#84] [Ideas]
Hi Openkat project,

I would like to have the option to perform bulk actions on grids.
For example to set the Clearance level on all selected items, or add several items to a network.

If I am overlooking the feature, please point me into the right direction.

Thanks in advance!

Best regards, Erik

**Comment by underdarknl** [comment]
This is being build as we speak:
https://github.com/minvws/nl-kat-rocky/pull/86 and https://github.com/minvws/nl-kat-rocky/pull/83
